src/renderer/components/Modal/DeleteModal.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { ModalBackground, ModalContainer } from "@public/style/Modal.styles";
+import ButtonBox from "@components/common/ButtonBox";
+
+const DeleteModal = ({ message, onConfirm, onClose }) => {
+  const handleConfirm = () => {
+    if (onConfirm) {
+      onConfirm();
+    }
+    onClose();
+  };
+
+  return (
+    <ModalBackground onClick={onClose}>
+      <ModalContainer className="large" onClick={e => e.stopPropagation()}>
+        <p>{message}</p>

```

Comment:
기존 Modal과 다른 사용성으로 인하여 새로 Modal component 추가하신것 좋은 것 같습니다 👍 
공통 컴포넌트의 스타일이 있기에,
아래의 태그들로 변경하시면 자동적으로 Modal의 UI가 통일되게 보일것 같습니다!

```suggestion
 <div className="title">
  <span>{message}</span>
</div>
```

Reply:
공통 스타일에 대한 부분을 고려하지 못한것 같습니다.😢
수정하여 반영하겠습니다.